doorkeeper-orms.gemspec

```diff
@@ -10,17 +10,23 @@ Gem::Specification.new do |s|
   s.authors     = ["jasl"]
   s.email       = ["jasl9187@hotmail.com"]
   s.homepage    = "http://github.com/doorkeeper-gem/doorkeeper-orms"
-  s.summary     = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper."
-  s.description = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper"
+  s.summary     = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
+  s.description = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
```

Comment:
I address those concerns in the [`load_doorkeeper` task](https://github.com/doorkeeper-gem/doorkeeper-orms/pull/2/files#diff-52c976fc38ed2b4e3b1192f8a8e24cffR7). It's far from elegant I think, and it has the problem of ORM specifics in doorkeeper's tests, but there is the single canonical test suite, that should run on each ORM extension repository through a clone (they wouldn't have much -if any- specs of their own).